crates/rate_limit/src/token_bucket.lua

```diff
@@ -6,39 +6,24 @@ local refillRate = tonumber(ARGV[3]) -- how many tokens are refilled after each
 local now = tonumber(ARGV[4]) -- current timestamp in milliseconds
 
 local results = {}
-
-for i, key in ipairs(keys) do
+for i, key in ipairs(KEYS) do
```

Comment:
☹️ I would prefer if we touched the Lua script as minimally as possible. It's a direct copy of the linked script above with only the tweaks necessary to make it work with multiple keys. You may have introduced bugs and made it harder to keep in-sync with the implementation above. Do you really think you've made significant performance improvements worth risking these changes?

Reply:
Reverted it in b9ecde904124b206b54d37e54f7b5672f3881fda if you find it hard to debug the minified version of the Lua script since it's not a huge optimization. Let's stick to the initial one and move forward with it.